### DISCUSSION TITLE
New Website

### DISCUSSION THREAD

**Discussion by ethanaobrien** [EmulatorJS/EmulatorJS#401] [General]
Hello everyone! I am excited to announce that I have officially gotten and set up a website! The new, official website is https://emulatorjs.org/ . Please note, since the website is brand new not every dns server will have the dns settings ready. This may take several refreshes (or just a couple of days) for this to populate!

**Comment by allancoding** [comment]
I with redirect emulatorjs.ga to https://emulatorjs.org/
Question when we get netplay working will we host a public server?

**Reply by ethanaobrien** [reply]
>Question when we get netplay working will we host a public server?

Hopefully, I plan to offer the new netplay server in nodejs and dotnet. Although heroku is no longer going to be free as of next month so that might be an issue

**Reply by allancoding** [reply]
I finished the redirect! Test: https://emulatorjs.ga/editor.html

**Reply by allancoding** [reply]
I would use cloudflare for your name server because all of the setting you can set. With security and Analytics and Caching etc.

**Reply by ethanaobrien** [reply]
Yes, I am using Cloudflare for my name server

**Reply by allancoding** [reply]
Ha sorry I just noticed you just have google sign your certificate. Look at these nodejs hosting: Render (free tier), Vercel (hobby free forever plan), and Cyclic.

**Reply by Ramaerel** [reply]
If hosting's an issue, I have a Linux server with Linode doing essentially nothing outside of some minimal web server stuff for me. If it meets whatever requirements there might be I'd be happy to help

**Reply by ethanaobrien** [reply]
Awesome, thank you, I'll reach out to you if needed
